mappings/net/minecraft/class_6016.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_6016
```

Comment:
This one should be removed.

Reply:
Also shouldn't have been commited. Why tf is this here>?